web/template/watch.gohtml

```diff
@@ -200,18 +200,18 @@
             <div class="flex justify-between align-middle mt-2 md:hidden">
                 {{template "actions" . }}
             </div>
-            {{if $stream.Description}}
-                {{$isMinLen := gt (len $stream.Description) 128}}
-                <div x-data="{less: {{$isMinLen}}}" class="text-3 p-4 border-t-2 my-3 border-secondary video-description">
-                    <span class="font-semibold">Video Description:</span>
-                    <div :class="less && 'truncate-children'">{{.Description}}</div>
-                    <template x-if="{{$isMinLen}}">
-                        <button class='text-xs text-gray-500 dark:text-gray-400 uppercase mt-2'
-                                @click="less = !less" x-text="less ? 'show more' : 'show less'">show less
-                        </button>
-                    </template>
+            <div x-data="{descriptionlen: `{{len .Description}}`, less:true}"
+                 class="text-3 p-4 border-t-2 my-3 border-secondary video-description">
+                <span class="font-semibold"
+                      x-show="description.length>0">{{if .Description}}Video Description:{{end}}</span>
+                <div :class="less && 'truncate-children'"
```

Comment:
Nit: When there are new lines or paragraphs in the description, every paragraph gets truncated and still appears. We probably want to cut off the message independent of the formatting.

Reference:
![image](https://user-images.githubusercontent.com/30267166/161105077-18d6e8cd-d5f4-45dc-af93-e1ff91a26eee.png)

Reply:
I noticed that to, but as it's the current behavior as well and not caused by this pr I'd make a separate issue for this.